src/sql/parts/objectExplorer/viewlet/treeNodeContextKey.ts

```diff
@@ -15,11 +15,13 @@ export class TreeNodeContextKey implements IContextKey<TreeNode> {
 	static SubType = new RawContextKey<string>('nodeSubType', undefined);
 	static Status = new RawContextKey<string>('nodeStatus', undefined);
 	static TreeNode = new RawContextKey<TreeNode>('treeNode', undefined);
+	static NodeLabel = new RawContextKey<string>('nodeLabel', undefined);
```

Comment:
I still think overall nodePath has value too. But I guess that can wait for a future PR.